src/MTGCore/ViewModels/SearchFilter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace MTGCore.ViewModels
+{
+    public class SearchFilter
+    {
+        public string Name { get; set; }
+        public string Type { get; set; }
+        public string Rarity { get; set; }
+        public string Set { get; set; }
+        public string Price { get; set; }
```

Comment:
What does price represent? Would it be worth making it an int?

Reply:
Price of a card, and yes would probably make more sense as a double actually.

Reply:
or create a new type called money or something